static/styles/portico/markdown.css

```diff
@@ -310,6 +310,8 @@
     pre {
         line-height: 15px;
         code {
+            font-size: 1.12em;
```

Comment:
How did you pick 1.12em?  

@amanagr FYI in case you have a suggested value for this.

(Also, does the `line-height` need to change in a corresponding way?)

Reply:
I did not suggest a value for this, but 16px looks like a good enough value for both line-height and font-size. 1.12rem (~18px) is excessive..

Reply:
16px seems reasonable to me.

Reply:
@timabbott I had chosen 1.12em as it corresponded to 18px AFAIR. 16px does look better, and i have modified the PR.